rust/main/agents/relayer/src/msg/op_queue.rs

```diff
@@ -115,153 +115,11 @@ impl OpQueue {
 #[cfg(test)]
 pub mod test {
     use super::*;
-    use hyperlane_core::{
-        HyperlaneDomain, HyperlaneMessage, KnownHyperlaneDomain, PendingOperationResult,
-        TryBatchAs, TxOutcome, H256, U256,
-    };
-    use serde::Serialize;
-    use std::{
-        collections::VecDeque,
-        time::{Duration, Instant},
-    };
+    use hyperlane_core::{HyperlaneDomain, KnownHyperlaneDomain};
+    use hyperlane_test::{mocks::MockPendingOperation, setup::dummy_metrics_and_label};
+    use std::collections::VecDeque;
     use tokio::sync;
 
-    #[derive(Debug, Clone, Serialize)]
-    pub struct MockPendingOperation {
```

Comment:
would avoid unnecessary code shift if possible, the OpQueue is only used by the relayer so it makes sense to leave here